recipe/activate-gcc.sh

```diff
@@ -145,7 +147,8 @@ _tc_activation \
   "DEBUG_CPPFLAGS,${DEBUG_CPPFLAGS:-${DEBUG_CPPFLAGS_USED}}" \
   "DEBUG_CFLAGS,${DEBUG_CFLAGS:-${DEBUG_CFLAGS_USED}}" \
   "CMAKE_PREFIX_PATH,${CMAKE_PREFIX_PATH:-${CMAKE_PREFIX_PATH_USED}}" \
-  "_CONDA_PYTHON_SYSCONFIGDATA_NAME,${_CONDA_PYTHON_SYSCONFIGDATA_NAME_USED}"
+  "_CONDA_PYTHON_SYSCONFIGDATA_NAME,${_CONDA_PYTHON_SYSCONFIGDATA_NAME_USED}" \
+  "CONDA_BUILD_SYSROOT,${CONDA_BUILD_SYSROOT:-${CONDA_BUILD_SYSROOT_USED}}"
```

Comment:
Can you check the following?
```suggestion
  "CONDA_BUILD_SYSROOT,${CONDA_PREFIX}/@CHOST@/sysroot"
```

Reply:
With the above, the conda build was successful and the tests passed ok.
Shall I switch to it?

Reply:
Yes

Reply:
Done. Locally it works, hopefully also in CI. So, resolving this.